pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,11 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) { return newGCECloud(config) })
 }
 
+// Raw access to the underlying GCE service, probably should only be used for e2e tests
```

Comment:
Sorry for the follow up comments. The risk of leaking the interface is that someone will forget to do something like `WaitForGlobal/RegionalOp`, or some other gce boilerplate, which will manifest as weird races till they realize.